browser.js

```diff
@@ -1,35 +1,40 @@
+var generatePrime = require('./generatePrime');
 var primes = require('./primes.json');
+
 var DH = require('./dh');
-var generatePrime = require('./generatePrime');
 
-exports.DiffieHellmanGroup =
```

Comment:
so I liked putting the exports up top because when you put them at the bottom of the file it can be very easy to miss them. Not a big deal but that was my reasoning.